Sources/StreamChat/Config/StreamRuntimeCheck.swift

```diff
@@ -13,7 +13,7 @@ public enum StreamRuntimeCheck {
     /// For *internal use* only
     ///
     ///  Enables background mapping of DB models
-    public static var _isBackgroundMappingEnabled = false
+    public static var _isBackgroundMappingEnabled = true
```

Comment:
Now we can remove the `_` since this won't be a "private" feature anymore:
```suggestion
    public static var isBackgroundMappingEnabled = true
```

Reply:
Would be a breaking change, also let's have the flexibility to turn it off if something goes wrong. Therefore, would prefer if we keep it.

Reply:
That is why `_` exists. It is saying that this is for now a private API, and once it is public, it ofc will break

Reply:
But what changed for it to be "public" API? It was always public, we are just turning this feature flag on by default.

Reply:
It was not public. It was internal, and only some customers were test-driving this. That is why we have this:
` For *internal use* only`.

The `underscore` is also a way to comunicate that it is a "internal" property and a breaking change can happen. This pattern is normally used across APIs and SDKs, even Apple uses this as well.

Reply:
Otherwise, what is the point of having the underscore in the first place?

Reply:
I think we have a misunderstanding about this one - we said that we **will turn it on by default**, not that we will make it public API so customers can change this as they see fit. That's what the PR title also says.

Reply:
I expect that we delete this key in coming months if it is stable for everyone. Renaming the key would trigger build errors for ones who were test-driving it first.